geode-core/src/main/java/org/apache/geode/management/DiskStoreMXBean.java

```diff
@@ -196,6 +196,11 @@
    */
   float getDiskUsagePercentage();
 
+  /**
+   * Returns the free percentage of the disk storage
```

Comment:
Add a "@since geode ..." tag.
Also, should you describe here when it will return -1?
I know you inherited "of the disk storage" from getDiskUsagePercentage but I'm not sure what that means. I think this mxbean represents just one disk store. If that is correct, then if these methods said "of this disk store" I think it would be clearer.